iroha/src/lib.rs

```diff
@@ -47,7 +47,8 @@ pub struct Iroha {
     sumeragi: Arc<RwLock<Sumeragi>>,
     kura: Arc<RwLock<Kura>>,
     transactions_receiver: Arc<RwLock<TransactionReceiver>>,
-    blocks_receiver: Arc<RwLock<BlockReceiver>>,
+    wsv_blocks_receiver: Arc<RwLock<BlockReceiver>>,
```

Comment:
Sorry for the late comment, but may we change naming of fields and aliases with aliases signature to receive blocks in different states? Instead of KuraBlockReceiver we can define SignedBlockReceiver = Receiver<SignedBlock>?